app/assets/stylesheets/events.scss

```diff
@@ -67,7 +67,7 @@
     text-align: center;
   }
 
-  tr td div.event-info {
+  tr td div.info-cell {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 4